internal/dependencytrack/policy.go

```diff
@@ -56,12 +56,39 @@ type ViolationAnalysis struct {
 	IsSuppressed  bool   `json:"isSuppressed,omitempty"`
 }
 
+// GetAllViolations returns violations for the entire portfolio. Suppressed
+// violations are omitted unless suppressed is true
+func (c *Client) GetAllViolations(suppressed bool) ([]*PolicyViolation, error) {
+	// dependency track per default only returns a 100 items per get, therefore we need to iterate over all PolicyViolation pages to get all PolicyViolation
+
+	// all PolicyViolation found in pagination
+	allPolicyViolations := []*PolicyViolation{}
+	// the last project found in the last pagination page result
+	lastPaginationPage := 1
+	// state var to show if all PolicyViolation projects where found
+	foundAll := false
```

Comment:
```suggestion
// GetAllViolations returns violations for the entire portfolio. Suppressed
// violations are omitted unless suppressed is true
func (c *Client) GetAllViolations(suppressed bool) ([]*PolicyViolation, error) {
	allPolicyViolations := []*PolicyViolation{}
	lastPaginationPage := 1
	foundAll := false
```

nit: I don't think the comments are necessary. I think it's clear enough from the code.